lib/cc/engine/bundler_audit/result.rb

```diff
@@ -0,0 +1,85 @@
+module CC
+  module Engine
+    module BundlerAudit
+      class Result
+        GEM_REGEX = /^\s*(?<name>\S+) \([\d.]+\)/.freeze
+        SEVERITIES = {
+          high: "critical",
+          medium: "normal",
+          low: "info",
+        }.freeze
+
+        def initialize(result, gemfile_lock)
+          @gem = result.gem
+          @advisory = result.advisory
+          @gemfile_lock = gemfile_lock
+        end
+
+        def to_issue
+          {
+            categories: ["Security"],
```

Comment:
FWIW: I think our rubocop config uses the default which recommends `%w` only at a certain size (2, 3?). If we intend to use it at any size we should update that config.